packages/ui/src/components/vuestic-components/va-tabs/VaTabs.vue

```diff
@@ -14,7 +14,7 @@
         flat
         size="medium"
         :icon="$props.prevIcon"
-        @click="movePaginationLeft()"
```

Comment:
It's preferable to keep brackets if you don't pass anything to method.
And remove them if you pass event.